catalyst_voices/lib/widgets/separators/voices_divider.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+const _kDefaultIntent = 24.0;
+
+/// Simple wrapper on Material [Divider].
+class VoicesDivider extends StatelessWidget {
+  final double indent;
+  final double endIntent;
+
+  const VoicesDivider({
```

Comment:
I noticed some widgets are better documented than others. For some you added  docs for fields, for some not. Do you think it would be worth to unify it?

Reply:
Even if adding something like:

```
/// See [Divider.indent].
final double indent;
```

Reply:
Yup. will try to add some more comments